src/cnaas_nms/scheduler/wrapper.py

```diff
@@ -33,6 +33,7 @@ def wrapper(job_id: Optional[str]=None, *args, **kwargs):
                 kwargs['kwargs']['job_id'] = job_id
         if job:
             job.start(fname = func.__name__)
+        kwargs['kwargs']['job'] = job
```

Comment:
Did this pass through integration test/wsgi container with mule? I'm not sure you can send a custom object/class to the mule since it's some kind of IPC with only text fields iirc

Reply:
I've tested it in the lab environment which runs Docker, uwsgi, mules etc. That seems to work. I can try to run it in the integration tests as well if you want to.

Reply:
Ah right, my bad. It's probably that the Jobtracker object and stuff in the wrapper is only executed inside the mule worker after the job has been triggered to run

Reply:
But if we pass the entire Job object it feels kind of silly to also pass the job_id as a separate argument? We could probably remove the job_id argument and use instead use job.id instead, what do you think?

Reply:
I see no reason to pass both. However, that must have been added after I did this. I'll fix it.